composer.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "doctrine/phpcr-dbal",
+  "name": "doctrine/phpcr-dbal-symfony-pack",
   "description": "Meta repository to install dbal implementation of doctrine/phpcr-bundle.",
-  "type": "library, metapackage",
+  "type": "library",
```

Comment:
i have no clue what is behind this one here, what we should use here. at the end it looks more like a `metapackage`